### DISCUSSION TITLE
Wordpress vs Django CMS for a blog application

### DISCUSSION THREAD

**Discussion by noorwali111** [django-cms/django-cms#7017] [Q&A] [answered]
Kindly answer me that what is best for blogging [Wordpress](https://tutorialform.com/how-to-login-wordpress-admin-panel/)  or django cms.

Thanks
Noor Wali Baz

**Comment by benzkji** [comment]
if you just wann blog, go with wordpress, honestly. If you are used to django, and dont like the "click and go and forget and have trouble with the plugins later" mode, you might wanna try django-cms. Honestly, I dont know which blogging module to recommend...there are a few. To find out, you'll need to try both ;-)

also:
https://www.django-cms.org/en/blog/2021/06/10/django-cms-vs-wordpress/

**Reply by olariuromeo** [reply]
It's like comparing an elephant to a louse, django cms is just one of thousands of modules in django framework nothing more. For those who ask themselves this question, it is better to use wordpress.

**Comment by careercoder** [comment]
It has many major differences

1.) The community of wordpress is by far greater than DjangoCMS 
2.) The learning curve to get things done is lower for wordpress than django
3.) While, I believe DjangoCMS is on the right path, it does have some missing features that are provided right out of the gate.

I would recommend DjangoCMS if you are building your own application, and not focusing on a blog. 
I add the caveat that djangocms will require you to do more actions if you are trying to add functionality than wordpress, but this also greatly depends on what your goals are.

**Comment by benzkji** [comment, marked as answer]
for me, the fact that "django is the foundation of it" is the main 
benefit. if that one isnt a point for you, you might as well be able to 
build the same thing (whatever thing.. ) with wordpress. though, if 
you'll have a bit more complex of a thing, django (with or without the 
cms ;-) will start to really shine. you'll develop in a DRY way, and 
you'll be a happier developer... ;-)


On 15.08.21 16:20, Jody Fitzpatrick wrote:
> It has many major differences
> 
> 1.) The community of wordpress is by far greater than DjangoCMS
> 2.) The learning curve to get things done is lower for wordpress than django
> 3.) While, I believe DjangoCMS is on the right path, it does have some 
> missing features that are provided right out of the gate.
> 
> I would recommend DjangoCMS if you are building your own application, 
> and not focusing on a blog.
> I add the caveat that djangocms will require you to do more actions if 
> you are trying to add functionality than wordpress, but this also 
> greatly depends on what your goals are.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/django-cms/django-cms/discussions/7017#discussioncomment-1186794>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AADRH6YOBORQW5JB7I6OH7LT47EJBANCNFSM42WFFJMA>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.
> 

-- 
Ben Stähli
***@***.***
+41 22 575 25 77
bnzk.ch
--
